### ISSUE TITLE
List over active tickets

### ISSUE THREAD

**Issue by JohanThomsen** [f-klubben/stregsystemet#271]
It would be nice to have a list over active tickets that you have bought (Such as f-acking, Flan etc.), visible on the user page.. If possible even have a small description about the event (date). The ticket should then be removed once the event is finished.

**Comment by hrjakobsen** [comment]
Great idea! It could be implemented with a model like this: 

```python
class Event(models.Model)
    product = models.ForeignKey(Product)
    time = models.DateTimeField(null=False,blank=False)
    ticket_start_threshold = models.DateTimeField(null=False,blank=False)
    ticket_end_threshold = models.DateTimeField(null=False,blank=False)
``` 

And then on the user page check if there is any purchases of a product within active event thresholds that matches the product of the event. As it can all be done in a single database query it would be super quick.
